### ISSUE TITLE
Categorical and Gamma measures

### ISSUE THREAD

**Issue by gdalle** [cscherrer/MeasureTheory.jl#145]
Hey, it's me again :wave: 
In order to replace the `Distributions.jl` dependency in `PointProcesses.jl` with `MeasureTheory.jl`, I would need local implementations of the Gamma and Categorical distributions. Is that on the wish list somehow?
I'm not very confident with the macros defining `ParametrizedMeasure`s, so I could give it a shot in a PR but I'm not sure it would land on target.

**Comment by cscherrer** [comment]
Yes! If you'd like to take a crack at it I can help if you get stuck, and of course review the PR. It would be really helpful to better understand the sticking points for this sort of thing.

I think I'd start by having a look at some of the existing parameterized measures to get a feel for it. I'm happy to help if something's not clear :)
